toolkit/imageconfigs/marketplace-gen1.json

```diff
@@ -22,6 +22,12 @@
                 {
                     "ID": "rootfs",
                     "Start": 9,
+                    "End": 999,
+                    "FsType": "ext4"
+                },
+                {
+                    "ID": "data",
```

Comment:
nit - consider using a more descriptive name than "data" for the "ID" for this new 500MB boot partition

Reply:
I'd second that. We generally organize our disks as:
- efi
- boot
- misc-fixed-size

Then finish with one of two options with "End": "0":
- rootfs
of
-fixed_size_rootfs
-data

Feels odd to have a boot partition at the end, and especially dynamically sized. Cloutd-init also often tries to expand the rootfs to fill the VM, and it can only do that if its the last partition

Reply:
I've always used `boot` and `efi` for the two nested boot partitions.

Reply:
Thanks Chris and Daniel. I have renamed the new partition to boot